renv/activate.R

```diff
@@ -0,0 +1,933 @@
+
```

Comment:
This file is auto-generated when creating an renv, so I'd not change it. This file is sourced in `.Rprofile`, it is bootstrapping renv and loading it.

Reply:
Hmm, I see. But I think we need some indication somewhere that this file is auto-generated. Otherwise, people will be confused as I was and start reviewing it or so.

Reply:
> some indication somewhere

seems easy to miss, any concrete ideas where to put that?

Reply:
Put autogenerated files in a folder `autogenerated`. That way you don't have to touch the file itself, and it is hard to miss.

Reply:
Actually the whole `renv` folder is auto-generated and completely managed by renv, so it might make things confusing for experienced renv users if we relocate this. I'm still in favor of just leaving this as it is

Reply:
But, the rest of the `renv` folder is not added to git, right? Having autogenerated stuff around is common, having it checked into version control is not.

Reply:
I mean, it is not a huge issue, I would also be fine with it as-is, as long as this file changes almost never, so it doesn't pollute diffs.